src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,111 @@
 /* eslint-disable */
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import { useAppSelector, useAppDispatch } from '../../app/store';
+import { loadedTodos } from '../../features/todos';
+import { currentTodoSlice, isSelected } from '../../features/currentTodo';
```

Comment:
Verify that `currentTodoSlice` contains the `setCurrentTodo` action. If this action is missing, it will cause errors when dispatching actions.